client/client.go

```diff
@@ -81,3 +82,10 @@ func NewClient(namespace string, context string, kubeConfigPath string) (*VanCli
 
 	return c, nil
 }
+
+func GetIngressRouteIfPossibleLoadBalancerIfNot(cli *VanClient) string {
```

Comment:
This appears to only be used by tests. If ingress is correctly defaulted on openshift versus k8s (see below) is this still needed?

Reply:
One thought would be to make this a method on the client interface, e.g. GetDefaultIngress(). That decision requires access to the client to determine if routes are supported. That method could then be used to set the default in the CLI.

Reply:
(see below answer) :-)

Reply:
The intention was, "not to determine a default anymore", IIUC, we have now three options:
-- ingress none
-- ingress  loadbalancer
-- ingress route

The default for the cli is just one of those three. No magic.

This helper function is used (for now) only for tests. Because that is the case exactly for tests... The test just wants to use "route or loadbalancer", the available one....
(more comments below)

* making this function a method... (I like that idea!) ... It would be something like "GiveMeTheRecommendedIngressInThisCase" -> aka: GetDefaultIngress...

Reply:
Well, now I think, if the intention is to have a double default on cli --ingress(route or loadbalancer). I think I will use this function/method for that...